benchmarks/benchmark_functions/qaoa.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Benchmarks for QAOA optimizations.
+"""
+import pennylane as qml
+from pennylane import numpy as np
+from .default_settings import _qaoa_defaults
+
+
+def benchmark_qaoa(hyperparams={}):
+	"""
```

Comment:
Curiously we seem to be using tabbed indents in the repo rather than 4 spaces. Don't think it's a problem though.

Reply:
Sounds like we just need to set up black :)